src/clarity-angular/tabs/_vertical-tabs.clarity.scss

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2016 - 2017 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import "../utils/variables.clarity";
+@import "../utils/mixins.clarity";
+@import "../utils/helpers.clarity";
+@import "../color/color.clarity";
+@import "../typography/typography.clarity";
+@import "../images/icons.clarity";
+
+
+.vtabs {
+    display: flex;
+    flex-direction: row;
+
+    &.is-reversed {
+        flex-direction: row-reverse;
+    }
+
+    &-nav {
+        flex: 0 180px;
+        padding-right: 12px;
+    }
+
+    &-content {
+        flex: 1;
+    }
+
+    .nav-group {
+        $navlink-padding: 4px;
```

Comment:
Users may want to customize these values so we might need to set them as default values.